docs/ota-client-guide/modules/ROOT/pages/libaktualizr-why-use.adoc

```diff
@@ -0,0 +1,25 @@
+= Integrate libaktualizr into your solution
+
+////
+This topic is supposed to outline the main use cases the product aims to address. The body of the guide must show how to use the product to implement these use cases.
+
+For libaktualizr We already have this topic: https://docs.atsgarage.com/client-config/advanced-update-control-with-libaktualizr.html
+The following text was taken from the linked topic and is a proposal for the introcdution to the integration guide.
+
+Feel free to adapt it or leave as-is. 
+////
+
+The OTA Connect client (aktualizr) is designed to be run as a standalone component on an embedded system and can manage the entire software update process. However, most automotive production use cases will have requirements that go beyond what the standalone client can provide. For example, some in-vehicle interfaces are proprietary and under NDA, so their implementation must be kept separate from aktualizr.
+
+= Why Use libaktualizr?
```

Comment:
Should "Use" be in lower case?

Reply:
Yes indeed it should be. This problem is that I have tons of micro-issues like this that I'm tracking in [a spreadsheet.](https://hmcs-my.sharepoint.com/:x:/g/personal/merlin_carter_here_com/EU2mLXZCq8lDiZJue5n3kJMB1yzokAsbEIw5ocIdH8ggIg?e=nOxqTf)

..so I'm trying to fix them in batches and reduce the number of commits. I've added it to the list

Reply:
fixed now